src/table/header-cell/th-element.tsx

```diff
@@ -66,6 +68,7 @@ export function TableThElement({
     getClassName: props => getStickyClassNames(styles, props),
   });
 
+  console.log('TH: tableVariant', tableVariant);
```

Comment:
remove console.log

Reply:
Thanks for spotting out. Fixed in [this commit](https://github.com/cloudscape-design/components/pull/3596/commits/14f0804174c7e0c7ebc55bfcf6d531c6c92303c6)